### PR TITLE
U026: Adding branding to the product

### PR DESCRIPTION
**Pull request by mtslech**
### 🛠 Changes being made

Here give examples of the changes you've made in this pull request. Include an itemized list if you can. It'll help the reviewer

### ✨ Definition of Done (all the points need to be filled out before merging)

- [x] Meets Acceptance Criteria
- [ ] Tested
- [ ] Code Review made by team 2 members
- [ ] UX approved
- [ ] Product Owner Approved

### 🧪 Non-Functional Requirements

- [x] Automated tests should cover acceptance criterias (Unit tests, Integration/API/Component tests)
- [x] All input must be validated (client side & API)
- [x] Sensitive data should not be stored in the URL
- [x] No secrets in codesource

### CODE REVIEW

File: product-web-app/common/req.express.session.mock.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const example = {
   isLoggedIn: true, /// to be changed for req.express.session.token
-  role: 'admin',
+  role: '2',
 };
 
 export default example;
```
Comment (Fluffyguy611) [inline product-web-app/common/req.express.session.mock.ts]: No longer needed as the authorisation system will be implemented soon

File: product-web-app/views/parent.html [hunk 2]
```diff
@@ -57,36 +101,58 @@
                                 Available actions:
                             </a>
                             <ul class="dropdown-menu" aria-labelledby="navbarDropdown">
-                                {% if role == 'admin' %}
-                                <li><a class="dropdown-item" href="/admin/add-job-roles">Add New Job Role</a></li>
-                                <li><a class="dropdown-item" href="">Delete</a></li>
-                                <li><a class="dropdown-item" href="">Update</a></li>
-                                <li><a class="dropdown-item" href="/job-roles">View Job Roles</a></li>
-                                <li><a class="dropdown-item" href="/capability">View Capability Leads</a></li>
+                                {% if role.length > 0 %}
```
Comment (Fluffyguy611) [inline product-web-app/views/parent.html]: {% if isAdmin == "1" or isAdmin == "2" %}

File: product-web-app/views/parent.html [hunk 3]
```diff
@@ -57,36 +101,58 @@
...
+                                <li class="nav-item active">
+                                    <a class="logout-nav" aria-current="page" href="/auth/logout">Logout</a>
+                                </li>
+                                {% endif %}
+                                {% if role == "1" %}
```
Comment (Fluffyguy611) [inline product-web-app/views/parent.html]: {% if isAdmin == "1" %}

File: product-web-app/views/parent.html [hunk 4]
```diff
@@ -57,36 +101,58 @@
...
+                                    <li><a class="dropdown-item" href="/admin/add-job-roles">Add New Job Role</a></li>
+                                    <li><a class="dropdown-item" href="/job-roles">View Job Roles</a></li>
+                                    <li><a class="dropdown-item" href="/capability">View Capability Leads</a></li>
+                                {% elif role == "2" %}
```
Comment (Fluffyguy611) [inline product-web-app/views/parent.html]: {% elif isAdmin == "2" %}

File: product-web-app/views/parent.html [hunk 5]
```diff
@@ -57,36 +101,58 @@
...
+            <div class="login">
+                <a href="/auth/login"><button class="login-button">Login</button></a>
+            </div>
+            <div>
+                <img src="https://media.tenor.com/lx2WSGRk8bcAAAAC/pulp-fiction-john-travolta.gif" alt="">
+            </div>
+            <div class="register">
+                <a href="/auth/register"><button class="register-button">Register</button></a>
+            </div>
     </div>
```
Comment (Fluffyguy611) [inline product-web-app/views/parent.html]: delete this as this is now a separate file named "not_found" se my PR US041

File: product-web-app/views/parent.html [hunk 6]
```diff
@@ -57,36 +101,58 @@
...
+        <div class="footer__bottom">
+            <div class="footer__bg-image footer__bg-image--desktop"></div>
+            <div class="footer__container footer__container--image container">
+                <div class="footer__slogan">
+                    <b>Thinking</b><br>
+                    <span>beyond</span><br>
+                    <b>limitations</b>
+                </div>
+                    <div class="footer__copyrights">
+                        <b>Contact Us: dzikiadriana@kainos.com</b><br>
```
Comment (Fluffyguy611) [inline product-web-app/views/parent.html:150]: # NICE CAULK
[
image
](url)

**Review by Fluffyguy611 — APPROVED** [review]

File: product-web-app/common/req.express.session.mock.ts (hunk 1 above)
Comment (ljercha) [reply product-web-app/common/req.express.session.mock.ts]: to be updated after Authorization

**Review by ljercha — APPROVED** [review]

**Pull request merged**